src/main/java/com/google/plantasy/utils/GameUtils.java

```diff
@@ -86,16 +86,17 @@ public static boolean addUserToGame(String userId, Entity gameEntity, DatastoreS
         log.severe("found null datastore trying to game to user " + userId);
         return false;
     }
+
+    Game game = new Game(gameEntity);
  
-    ArrayList<String> userIds = (ArrayList<String>) gameEntity.getProperty("userIds");
+    ArrayList<String> userIds = game.getUserIds();
     if(userIds == null){
       userIds = new ArrayList<String>();
+      game.setUserIds(userIds);
```

Comment:
We set the user Ids but we don't update the datastore, so if the server gets refreshed or if we try to reference this again, we'll lose that data. We can call game.GetGameEntity() below and do datastore.put() with that as the param.

Reply:
That's done in all the places where an entity is updated.